src/PerfView/Themes/LightTheme.xaml

```diff
@@ -85,9 +85,9 @@ SOFTWARE.
     <SolidColorBrush x:Key="Menu.Static.Separator" Color="#FFD7D7D7"/>
     <SolidColorBrush x:Key="Menu.Disabled.Foreground" Color="#FF707070"/>
     <SolidColorBrush x:Key="MenuItem.Selected.Background" Color="#3D26A0DA"/>
-    <SolidColorBrush x:Key="MenuItem.Selected.Border" Color="#FF2084E8"/>
+    <SolidColorBrush x:Key="MenuItem.Selected.Border" Color="#FF1F4E79"/>
```

Comment:
This doesn't seem to work either - it looks like the high contrast theme overrides the highlighting color and chooses something very bright.  The bottom status bar in PerfView is properly themed such that under dark high-contrast themes, the text is light and the background is dark.  Can you please theme the top file menu bar in the same way?